pkg/beholder/message.go

```diff
@@ -166,6 +169,8 @@ func OtelAttr(key string, value any) otellog.KeyValue {
 		return otellog.Bool(key, v)
 	case []byte:
 		return otellog.Bytes(key, v)
+	case time.Time:
+		return otellog.Int64(key, v.Unix())
```

Comment:
In `beholder/client.go` we're setting `timestamp` to `time.Now().UTC()` which is a string representing a date time. Here however we're using the epoch timestamp. Or are these unrelated?

Reply:
UTC() returns a `time.Time` which can be parsed into a string or any kind of unix timestamp. I decided to use a unix timestamp here but we can render it as a string too, don't really have a preference.